good_bot/good_bot.py

```diff
@@ -24,13 +33,7 @@ def greet_new_user(update: Update, context: CallbackContext):
     update.message.reply_text(f"[{bot_message}] {greeting_message}, {new_user_name}!")
 
 
-def main():
-    # Set up argument parsing
-    parser = argparse.ArgumentParser(description='Autobot for Telegram')
-    parser.add_argument('--greeting', type=str, help='The greeting message to send to new users')
-    parser.add_argument('--bot_id', type=str, help='The bot ID for the Telegram bot')
-    parser.add_argument('--chat_id', type=str, help='The chat ID for the Telegram channel')
-    args = parser.parse_args()
+def server(args: Args):
```

Comment:
Nit: This refactor is just vibes, love how cli responsibility is decomposed into its own module 🔥🔥